climbing_stairs/climbing_stairs.py

```diff
@@ -2,13 +2,20 @@
 
 import sys
 
+
```

Comment:
How can we avoid repeating work and reduce the run time complexity to O(n) using memoization?